central/componentcveedge/datastore/internal/postgres/store.go

```diff
@@ -81,27 +77,34 @@ func (s *storeImpl) Count(ctx context.Context) (int, error) {
 func (s *storeImpl) Exists(ctx context.Context, id string, imageComponentId string, imageCveId string) (bool, error) {
 	defer metrics.SetPostgresOperationDurationTime(time.Now(), ops.Exists, "ComponentCVEEdge")
 
-	row := s.db.QueryRow(ctx, existsStmt, id, imageComponentId, imageCveId)
-	var exists bool
-	if err := row.Scan(&exists); err != nil {
-		return false, pgutils.ErrNilIfNoRows(err)
-	}
-	return exists, nil
+	var sacQueryFilter *v1.Query
+
+	q := search.ConjunctionQuery(
+		sacQueryFilter,
```

Comment:
umm, why are we adding `sacQueryFilter` when we know it is nil?

Reply:
Good question. I think the name of this PR was wrong, I'll fixt that. This change unify the generated code to not use SQL but `v1.Query` instead (look at removed lines at the top of the file). Previously we cannot have it (v1.Query) for multikey resources so I skipped them. SAC filter  is nil because we have not implemented SAC for indirectly scoped resources. It will be fixed in another PR [ROX-11159](https://issues.redhat.com/browse/ROX-11159)

Reply:
What is the current functionality if sacQueryFilter is nil in a conjunction query? Does this currently just make it not work?